source/web-service/flaskapp/routes/ingest.py

```diff
@@ -78,6 +78,16 @@ def ingest_get():
     return abort(response)
 
 
+# 'DELETE' method. Create workload and call existing method ('POST' with '_delete': True)
+@ingest.route("/<id>", methods=["DELETE"])
```

Comment:
I think we wanted to be able to delete a resource using the resource URI, not the ingest route.